### PR TITLE
refactor file locations

### PR DESCRIPTION
**Pull request by devoncarew**
This PR refactors some file locations; github wants a few files at the root of a repo (and I believe a previous refactor to convert this repository into a mono repo unintentionally moved those files to sub-directories).

- copy the protobuf/LICENSE file to the repo root
- move the protobuf/AUTHORS file to the repo root
- move the protobuf/CONTRIBUTING.md file to the repo root
- various minor changes to the readme files

### CODE REVIEW

**Comment by devoncarew** [comment]
Actually, we may want to delete the CONTRIBUTING.md file from this repo; if we do that, github will fall back to using the one for this github org - https://github.com/google/.github/blob/master/CONTRIBUTING.md.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+This repository is home to packages related to
+[protobuf for Dart](https://pub.dev/documentation/protobuf/latest/).
+
+## protobuf
+[protobuf](protobuf/) - a support library for the generated code
 
- - [protoc_plugin](protoc_plugin/) A Dart back-end for the protoc compiler.
- - [protobuf](protobuf/) Support library for the generated code.
+## protoc_plugin
+[protoc_plugin](protoc_plugin/) - a Dart back-end for the protoc compiler
```
Comment (osa1) [inline README.md:16]: "protobuf" and "protoc_plugin" should be smaller than the repo description I think. Maybe make "Protobuf support for Dart" h1, or "protobuf" and "protoc_plugin" h3?

I'm actually not sure if we need headers given that they don't have much text under, I like the current version. Is this a template we use for Google repos?

**Review by osa1 — APPROVED** [review]
Thanks!

Do we want to remove LICENSE files in `protobuf/` and `protoc_plugin/`?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Copyright 2013, the Dart project authors.
```
[the remark this answers is not in the document]
Comment (devoncarew) [reply LICENSE:1]: No, we generally leave the year alone here (just leave it as whatever is was initially at the start of the repo).

File: README.md (hunk 1 above)
Comment (devoncarew) [reply README.md:16]: There isn't really a standard we use for monorepos. I'm going to re-work this however to use a markdown table and see how that looks.

**Comment by devoncarew** [comment]
Here's the top-level readme refactored into a markdown table:

Screen Shot 2022-05-19 at 11 34 00 AM

**Review by kevmoo — APPROVED** [review]

**Comment by osa1** [comment]
The new README looks great!

**Pull request merged**